File: 441-arranging-coins/441-arranging-coins.cs
public class Solution {
    public int ArrangeCoins(int n) {
        double delta = n;
        delta = delta * 8 + 1;
        double res = (-1 + Math.Sqrt(delta)) / 2;
        int final_res;
        final_res = Convert.ToInt32(Math.Floor(res));
        return final_res;
    }
}

File: 441-arranging-coins/441-arranging-coins.java
class Solution {
    public int arrangeCoins(int n) {
        double delta = n;
        delta = 1 + 8 * delta;
        double res = ((-1 + Math.sqrt(delta)) / 2);
        int final_res = (int)res;
        return final_res;
    }
}